int32 handle
uint8 deviceRegister
---
bool hasSucceeded
uint16 value